test/jdk/java/util/zip/DataDescriptorSignatureMissing.java

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright 2012 Google, Inc.  All Rights Reserved.
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Yeah, but the original hunk has `2023`. Leave it as-as, i.e. at `2023`:
https://github.com/openjdk/jdk/commit/07eaea8c25bae6ed852685f082f8b50c5b20c1a9#diff-20b351f8e90d650865fb32bf9208e9a8a0f77ad2deefd38dbbf062b0cc88fb5cR3

Reply:
fixed!